config/apidoc.php

```diff
@@ -87,4 +87,16 @@
             ],
         ],
     ],
+
+    /*
+     * Custom logo path. Will be copied during generate command.
```

Comment:
Can you add an explanation for the default? Something like "Set this to false to use the default logo". Also, please update the documentation.